.github/actions/run-backend-tests/action.yml

```diff
@@ -122,7 +122,7 @@ runs:
           if: ${{ inputs.ee == 'true' && inputs.group == '1' }}
           shell: bash
           run: |
-              pytest posthog -m "ee" --durations=100 --durations-min=1.0
+              PERSON_ON_EVENTS_ENABLED=true pytest posthog -m "ee" --durations=100 --durations-min=1.0
```

Comment:
this should fail, just a smoke test that person_on_events tests are indeed running fine. Will revert back after CI fails